sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasing.scala

```diff
@@ -178,11 +195,16 @@ object NestedColumnAliasing {
             nestedFieldToAlias
               .map { case (nestedField, _) => totalFieldNum(nestedField.dataType) }
               .sum < totalFieldNum(attr.dataType)) {
-          Some(attr.exprId -> nestedFieldToAlias)
+          Some((attr.exprId, nestedFieldToAlias))
```

Comment:
Why did change this?

Reply:
https://github.com/apache/spark/pull/28898/files#diff-957112380b0a2ef014abc8227d0b70acR479-R496

Reply:
?

Reply:
It's related to the comment below. https://github.com/apache/spark/pull/28898#discussion_r447950809